rest/backend_server/Ballerina.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+org = "bff"
```

Comment:
can we make it `bff_samples` to be aligned with integration samples? https://github.com/ballerina-guides/integration-samples/blob/main/github-pull-requests-to-gsheets/Ballerina.toml#L2

Reply:
yes, I will update

Reply:
Updated. c15365d59e311c6748120203044dbfc6c4083a67